website/common/locales/en/settings.json

```diff
@@ -184,5 +184,25 @@
     "suggestMyUsername": "Suggest my username",
     "everywhere": "Everywhere",
     "onlyPrivateSpaces": "Only in private spaces",
-    "bannedSlurUsedInProfile": "Your Display Name or About text contained a slur, and your chat privileges have been revoked."
+    "bannedSlurUsedInProfile": "Your Display Name or About text contained a slur, and your chat privileges have been revoked.",
+    "transactions": "Transactions",
+    "gemTransactions": "Gem Transactions",
+    "hourglassTransactions": "Hourglass Transactions",
+    "noGemTransactions": "You don't have any gem transactions yet.",
+    "noHourglassTransactions": "You don't have any hourglass transactions yet.",
+    "transaction_debug": "Debug Action",
```

Comment:
Do these strings even need to be translated, given they don't face the end user?

Reply:
The `transaction_debug` one might not be, that's true. The other ones are user facing though in the transaction table when you don't have any transactions yet as an empty state.

Reply:
Although the debug action one is technically also user facing, since it is also shown in the UI. (although this would only happen on local dev instances)

Reply:
kk!